src/__tests__/proposals.int.test.ts

```diff
@@ -723,4 +740,167 @@ describe('/proposals', () => {
+		});
+
+		it('returns 400 bad request when no proposalId is sent', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 1,
+				})
+				.expect(400);
+			expect(result.body).toMatchObject({
+				name: 'InputValidationError',
+				details: expect.any(Array) as unknown[],
+			});
+		});
+
+		it('returns 400 bad request when no organizationId is sent', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					proposalId: 1,
+				})
+				.expect(400);
+			expect(result.body).toMatchObject({
+				name: 'InputValidationError',
+				details: expect.any(Array) as unknown[],
+			});
+		});
+
+		it('returns 409 Conflict when a non-existent organization is sent', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 42,
+					proposalId: 1,
+				})
+				.expect(409);
+			expect(result.body).toMatchObject({
+				name: 'DatabaseError',
+			});
+		});
+
+		it('returns 409 Conflict when attempting to create a duplicate OrganizationProposal', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			await createOrganizationProposal({
+				organizationId: 1,
+				proposalId: 1,
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 1,
+					proposalId: 1,
+				})
+				.expect(409);
+			expect(result.body).toMatchObject({
+				name: 'DatabaseError',
+			});
+		});
+
+		it('returns 409 conflict when the proposal ID in the URL does not match the proposal ID in the body', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 1,
+					proposalId: 2,
+				})
+				.expect(409);
+			expect(result.body).toMatchObject({
+				name: 'InputConflictError',
```

Comment:
Rather than me manually checking and asking about the names of the expected errors, can I just ask that you take a quick pass/verify that all of these names and that they match your intent (I.e., no copypasta slipped in)? Feel free to resolve this as soon as you've done so, no note necessary.